packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-loading.tsx

```diff
@@ -0,0 +1,67 @@
+import { h } from '@stencil/core';
+
+export const DialLoading = () => (
+  <svg width="100%" height="100%" viewBox="0 0 276 276" data-testid="loading" preserveAspectRatio="xMidYMin meet">
+    <symbol id="sym-octagon" stroke="black" viewBox="0 0 200 200">
+      <defs>
```

Comment:
I believe there is already a near identical loading widget, is there a reason to not just use that?

Reply:
The sc-loading-spinner component just includes this dynamic circle. Then this component includes three parts with the outer ring , this dynamic circle and text 'loading'.